Sources/Subs.php

```diff
@@ -4982,9 +4983,31 @@ function inet_dtop($bin)
 	if ($db_type == 'postgresql')
```

Comment:
Testing the value of `$db_type` is not a reliable way to know the value of `$bin`. You need to replace this test with something that tests the value of `$bin`.